src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue

```diff
@@ -93,10 +93,41 @@
 				</div>
 				<div class="row">
 					<div class="columns small-12 large-expand">
-						Lend button
+						<action-button
+							class="expandable-loan-card-action-button"
+							:loan-id="loan.id"
+							:loan="loan"
+							:items-in-basket="itemsInBasket"
+							:is-lent-to="loan.userProperties.lentTo"
+							:is-funded="isFunded"
```

Comment:
Once master is merged in, can we add `isFunded` to the props list to resolve the console error?